jbpm-console-ng-showcase/src/main/java/org/jbpm/console/ng/server/impl/DashbuilderBootstrap.java

```diff
@@ -106,10 +111,10 @@ protected void registerDataSetDefinitions() {
                 .uuid(HUMAN_TASKS_WITH_USER_DATASET)
                 .name("Human tasks and users")
                 .dataSource(JBPM_DATASOURCE)
-                .dbSQL("select  t.activationTime, t.actualOwner, t.createdBy, "
```

Comment:
I think you refer this commit
https://github.com/droolsjbpm/jbpm-dashboard/commit/f8f1823250f6db86242d2f7e8b7d847398ce7dfc